### PR TITLE
BETA

### PR DESCRIPTION
**Pull request by LiveChief**

### CODE REVIEW

**Comment by LiveChief** [comment]
working adding client feature.

File: wireguard-server.sh [hunk 1]
```diff
@@ -527,16 +527,100 @@ qrencode -t ansiutf8 -l L < $HOME/$CLIENT_NAME-wg0.conf
...
+    until [[ "$WIREGUARD_OPTIONS" =~ ^[1-4]$ ]]; do
+        read -rp "Select an Option [1-4]: " -e -i 1 WIREGUARD_OPTIONS
+    done
+        case $WIREGUARD_OPTIONS in
+            1)
+            echo "Tell me a new name for the client config file. Use one word only, no special characters. (No Spaces)"
+	    read -p "New client name: " -e NEW_CLIENT_NAME
+	    ;;
+            2)
+            echo "This is under development."
```
Comment (Leopere) [inline wireguard-server.sh]: Test it but do this
```BASH
cat wg.confg|grep start| awk '{ print $2 }'
read -p "type in clientid : " CLIENTID
read -p "Are you sure you want to remove $CLIENTID? y/n: " -n 1 -r
if [[ $REPLY =~ ^[Yy]$ ]]
then
   echo
   sed -i "/\#\# $CLIENTID start/,/\#\# $CLIENTID end/d" wg.confg
else
   echo
   echo "exiting"
fi
```

Comment (LiveChief) [reply wireguard-server.sh]: i have to do (Math Portfolio, French Makeup Work)
Sorry my dood i can't complete this tonight and will look into it tomorrow

Comment (LiveChief) [reply wireguard-server.sh]: I don't we should ask for a CLIENTID, i think we should display all the CLIENTID and let them choose.

Comment (Leopere) [reply wireguard-server.sh]: Yeah, that's what the first line does there.

Comment (Leopere) [reply wireguard-server.sh]: This is going to sound ridiculous but a friend convinced me to submit his suggested code which was a weird version of mine and it seems we're missing some details I'd added.

Comment (Leopere) [reply wireguard-server.sh]:
```BASH
cat /etc/wireguard/wg.conf|grep start| awk '{ print $2 }'
read -p "Out of the clients we found type what client you wish to remove: " -e CLIENT_TO_REMOVE
read -p "Are you sure you would like to remove $CLIENT_TO_REMOVE? : " -n 1 -r
if [[ $REPLY =~ ^[Yy]$ ]]
then
 sed -i "/\#\# $CLIENTID start/,/$CLIENTID end/d" /etc/wireguard/wg.conf
fi
```

Somewhere between this suggestion and the first one will be what you should aim for.

Comment (LiveChief) [reply wireguard-server.sh]: PULL, PLEASE.

**Pull request closed without merging**